src/sentry/templates/sentry/emails/issue_alert_base.html

```diff
@@ -121,6 +121,13 @@ <h3>
                       {% endif %}
                     </h3>
                   </div>
+                {% elif type == "transaction" %}
+                  <div class="event-type transaction">
+                    <h3>
+                      <a href="{% absolute_uri link %}">{{ issue_type|truncatechars:40 }}</a><br />
+                        <span class="event-subtitle">{{ subtitle|truncatechars:40 }}</span>
+                    </h3>
+                  </div>
                 {% else %}
                   <div class="event-type default">
```

Comment:
We were falling into this block before and I guess `event.title` is unknown and there was no transaction. I'm not sure why that is the case on prod when locally it rendered fine before my change in this PR, must be a difference in how we're generating perf issues locally.